src/utils/command-helpers.ts

```diff
@@ -59,30 +45,30 @@ export const USER_AGENT = `${name}/${version} ${platform}-${arch} node-${process
 /** A list of base command flags that needs to be sorted down on documentation and on help pages */
 const BASE_FLAGS = new Set(['--debug', '--http-proxy', '--http-proxy-certificate-filename'])
 
-export const NETLIFY_CYAN = chalk.rgb(40, 180, 170)
+// XXX(serhalp) Restore equivalent of `chalk.rgb(40, 180, 170)`
+export const NETLIFY_CYAN = picocolors.blueBright
 
-export const NETLIFYDEV = `${chalk.greenBright('◈')} ${NETLIFY_CYAN('Netlify Dev')} ${chalk.greenBright('◈')}`
-export const NETLIFYDEVLOG = `${chalk.greenBright('◈')}`
-export const NETLIFYDEVWARN = `${chalk.yellowBright('◈')}`
-export const NETLIFYDEVERR = `${chalk.redBright('◈')}`
+export const NETLIFYDEV = `${picocolors.greenBright('◈')} ${NETLIFY_CYAN('Netlify Dev')} ${picocolors.greenBright('◈')}`
+export const NETLIFYDEVLOG = `${picocolors.greenBright('◈')}`
+export const NETLIFYDEVWARN = `${picocolors.yellowBright('◈')}`
+export const NETLIFYDEVERR = `${picocolors.redBright('◈')}`
 
 export const BANG = process.platform === 'win32' ? '»' : '›'
 
 /**
  * Sorts two options so that the base flags are at the bottom of the list
- * @param {import('commander').Option} optionA
- * @param {import('commander').Option} optionB
- * @returns {number}
  * @example
- * options.sort(sortOptions)
+ * options.sort(compareOptions)
  */
-// @ts-expect-error TS(7006) FIXME: Parameter 'optionA' implicitly has an 'any' type.
-export const sortOptions = (optionA, optionB) => {
+export const compareOptions = (optionA: Option, optionB: Option): number => {
+  const longOptionA = optionA.long ?? ''
+  const longOptionB = optionB.long ?? ''
   // base flags should be always at the bottom
-  if (BASE_FLAGS.has(optionA.long) || BASE_FLAGS.has(optionB.long)) {
+  // FIXME(serhalp) This logic can't be right? Look at it... This must only happen to work.
+  if (BASE_FLAGS.has(longOptionA) || BASE_FLAGS.has(longOptionB)) {
     return -1
   }
```

Comment:
Reviewer, am I tripping? This should be something like this, right?
```suggestion
  if (BASE_FLAGS.has(longOptionA)) {
    return 1
  } else if (BASE_FLAGS.has(longOptionB)) {
    return -1
  }
```

Reply:
Your suggestion looks to be correct. A negative value indicates that `a` should come before `b`, and a positive value indicates that `a` should come after `b`.